src/components/apis/list-of-apis/ko/runtime/api-list-dropdown.html

```diff
@@ -14,11 +14,11 @@
             <!-- /ko -->
             <!-- /ko -->
         </div>
-        <button class="input-group-addon no-border" data-toggle="dropdown" aria-label="Expand APIs list">
+        <button class="dropdown-input-button" data-toggle="dropdown" aria-label="Expand APIs list">
             <i class="icon-emb icon-emb-chevron-down"></i>
         </button>
-        <div class="dropdown">
-            <input type="search" class="form-control form-control-light" aria-label="Search" placeholder="Search APIs"
+        <div class="dropdown drop-down">
```

Comment:
`drop-down` - maybe `dropdown-container` instead? Just looks a bit weird to have 2 "dropdown" classes here.